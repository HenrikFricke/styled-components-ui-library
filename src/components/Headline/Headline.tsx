import styled from 'styled-components';

import {
    colors,
    fonts
} from '../../constants';

export const H1 = styled.h1`
    color: ${colors.PRIMARY};
    padding: 10px;
    font-size: ${fonts.LARGE};
`;