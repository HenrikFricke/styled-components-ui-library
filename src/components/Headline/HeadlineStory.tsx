import * as React from 'react';
import { storiesOf } from '@storybook/react';

import { H1 } from './Headline';

storiesOf('Headline', module)
    .add('H1', () => (
        <H1>I'am a headline</H1>
    ));