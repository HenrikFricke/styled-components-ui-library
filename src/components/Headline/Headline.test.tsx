import * as React from 'react';
import ReactTestRenderer from 'react-test-renderer';

import 'jest-styled-components';

import { H1 } from './Headline';

describe('Headline', () => {
    let tree: ReactTestRenderer.ReactTestRendererJSON;

    beforeEach(() => {
        tree = ReactTestRenderer.create(
            <H1>I'm a healdine</H1>
        ).toJSON();
    });

    it('should render correctly', () => {
        expect(tree).toMatchStyledComponentsSnapshot();
    });
});