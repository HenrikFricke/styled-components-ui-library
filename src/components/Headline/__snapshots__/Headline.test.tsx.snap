// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline should render correctly 1`] = `
.jxAQPs {
  color: palevioletred;
  padding: 10px;
  font-size: 32px;
}

<h1
  className="sc-bdVaJa jxAQPs"
>
  I'am a healdine
</h1>
`;
